.spi.yml

```diff
@@ -0,0 +1,19 @@
+# This is manifest file for the Swift Package Index for it to 
+# auto-generate and host DocC documentation.
+# For reference see https://blog.swiftpackageindex.com/posts/auto-generating-auto-hosting-and-auto-updating-docc-documentation/.
```

Comment:
FWIW, the better link for this might be: https://swiftpackageindex.com/swiftpackageindex/spimanifest/0.13.0/documentation/spimanifest/commonusecases#Host-DocC-documentation-in-the-Swift-Package-Index

Reply:
Thanks! Amended.